src/directives/modal.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @fileoverview Provides the "ngeoModal" directive. This directive allows
+ * displaying a modal window. The directive takes care or the show/hide
+ * actions.
```

Comment:
Is the directive bootstrap-specific? If so it should be documented. Also, does the directive requires bootstrap.js and jQuery? If so it should be documented as well.